WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/AddOrderCoordinator.swift

```diff
@@ -86,3 +108,48 @@ private extension AddOrderCoordinator {
         ServiceLocator.analytics.track(event: WooAnalyticsEvent.Orders.orderAddNew())
     }
 }
+
+class BaseViewController: UIViewController {
```

Comment:
We can find a more meaningful name for this class

Reply:
Definitely! Renamed at aee2e57f3b3ae77dc7c6a4acffda42997787d7d0